pdf/src/crypt.rs

```diff
@@ -442,6 +442,10 @@ impl Decoder {
             return Ok(data);
         }
 
+        if data.len() == 0 {
```

Comment:
```suggestion
        if data.is_empty() {
```
I believe clippy advises this for readability. I agree! (API [docs](https://doc.rust-lang.org/std/primitive.slice.html#method.is_empty)).